LivingCity/traffic/b18CUDA_trafficSimulator.cu

```diff
@@ -1304,7 +1323,13 @@ void b18ResetPeopleLanesCUDA(uint numPeople) {
   cudaMemset(&laneMap_d[halfLaneMap], -1, halfLaneMap*sizeof(unsigned char));
 }
 
-void b18SimulateTrafficCUDA(float currentTime, uint numPeople, uint numIntersections, float deltaTime) {
+void b18SimulateTrafficCUDA(float currentTime,
+  uint numPeople,
+  uint numIntersections,
+  float deltaTime,
+  const parameters simParameters,
```

Comment:
It might be a good idea to pass this structure by reference, just in case it grows in the future.